KioskApplication/src/main/java/controller/map/PathView.java

```diff
@@ -0,0 +1,55 @@
+package controller.map;
+
+import database.objects.Edge;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Point2D;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.shape.Line;
+
+import java.io.IOException;
+
+public class PathView extends AnchorPane {
```

Comment:
This is doing the same thing as EdgeView. Is there a way you can either subclass edgeview and add functionality or just add the functionality to EdgeView?